pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -1380,4 +1380,37 @@ <h4>Contribute to "Understanding OData in 6 steps"</h4>
             modalBody.insertBefore(screenReaderContent, modalBody.firstChild);
         });
     });
+
+    function trapFocus(element) {
+        var focusableElements = element.querySelectorAll('a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type="text"]:not([disabled]), input[type="radio"]:not([disabled]), input[type="checkbox"]:not([disabled]), select:not([disabled])'),
+        firstFocusableElement = focusableElements[0];  
+        lastFocusableElement = focusableElements[focusableElements.length - 1];
+        KEYCODE_TAB = 9;
+
+        element.addEventListener('keydown', function(e) {
+            var isTabPressed = (e.key === 'Tab' || e.keyCode === KEYCODE_TAB);
+
+            if (!isTabPressed) { 
+                return; 
+            }
+
+            if ( e.shiftKey )
+            {
+                if (document.activeElement === firstFocusableElement) 
+                {
+                    lastFocusableElement.focus();
+                        e.preventDefault();
+                    }
```

Comment:
Why is `e.preventDefault()` indented? And why is there a closing brace directly below it?

Reply:
fixed